ddtrace/propagation/http.py

```diff
@@ -21,6 +23,7 @@
 from ..internal.constants import PROPAGATION_STYLE_B3
 from ..internal.constants import PROPAGATION_STYLE_B3_SINGLE_HEADER
 from ..internal.constants import PROPAGATION_STYLE_DATADOG
+from ..internal.constants import PROPAGATION_STYLE_W3C_TRACECONTEXT
```

Comment:
We don't do this for any of the other propagation styles constants in this module. I am fine with keeping the code as is for consistency. But in the future we should avoid exposing constants in our public api. 

```suggestion
from ..internal.constants import PROPAGATION_STYLE_W3C_TRACECONTEXT as _PROPAGATION_STYLE_W3C_TRACECONTEXT
```

Reply:
Good point. I agree this should be internal (as is `_TraceContext`) but this aliasing is cumbersome if we were going to standardize on it. Hopefully this improves with refactoring of this module, i.e., split the file into file per propagator and this module only exposes the public interface for use in manual instrumentation.